lib/beaker-pe/install/pe_utils.rb

```diff
@@ -450,7 +450,15 @@ def do_install hosts, opts = {}
               else
                 prepare_host_installer_options(host)
                 register_feature_flags!(opts)
-                generate_installer_conf_file_for(host, hosts, opts)
+
+                if opts[:upgrade] && use_meep?(host['previous_pe_ver'])
+                  # merge answers into pe.conf
+                  update_pe_conf(opts[:answers])
+                  update_pe_conf(opts[:custom_answers])
```

Comment:
I would think it would make sense to do it right here? I should probably add a condition that only does `update_pe_conf` on the master, and then SCPs the pe.conf over on any other node.